src/LazyOperations/InverseLinearMap.jl

```diff
@@ -0,0 +1,333 @@
+    @assert dim(Z) == size(M, 2) "a linear map of size $(size(M)) cannot " *
+            "be applied to a set of dimension $(dim(Z))"
+    return ZeroSet{N}(size(M, 1))
+end
+
+# EmptySet is absorbing for LinearMap
+function InverseLinearMap(M::AbstractMatrix, ∅::EmptySet)
+    return ∅
+end
+
+
+# --- AbstractAffineMap interface functions ---
+
+function matrix(ilm::InverseLinearMap)
+    return ilm.M
+end
+
+function vector(ilm::InverseLinearMap{N}) where {N}
+    return spzeros(N, dim(ilm))
+end
+
+function set(ilm::InverseLinearMap)
+    return ilm.X
+end
+
+"""
+    dim(ilm::InverseLinearMap)
+
+Return the dimension of an inverse linear map.
+
+### Input
+
+- `ilm` -- inverse linear map
+
+### Output
+
+The ambient dimension of the inverse linear map.
+"""
+function dim(ilm::InverseLinearMap)
+    return size(ilm.M, 1)
+end
+
+"""
+    σ(d::AbstractVector{N}, ilm::InverseLinearMap{N}) where {N<:Real}
+
+Return the support vector of the inverse linear map.
+
+### Input
+
+- `d`   -- direction
+- `ilm` -- inverse linear map
+
+### Output
+
+The support vector in the given direction.
+If the direction has norm zero, the result depends on the wrapped set.
+
+### Notes
+
+If ``L = M^{-1}⋅S``, where ``M`` is a matrix and ``S`` is a convex set, since
+(M^T)^{-1}=(M^{-1})^T, it follows that ``σ(d, L) = M^{-1}⋅σ((M^T)^{-1} d, S)``
+for any direction ``d``.
+"""
+function σ(d::AbstractArray{N,1}, ilm::InverseLinearMap{N,S,NM,MAT}) where {N<:Real, S<:LazySet{N}, NM, MAT}
+    return ilm.M \ σ(transpose(ilm.M) \ d, ilm.X) #TODO: Check this formula
+end
+
+"""
+    ρ(d::AbstractVector{N}, ilm::InverseLinearMap{N}; kwargs...) where {N<:Real}
+
+Return the support function of the inverse linear map.
+
+### Input
+
+- `d`      -- direction
+- `ilm`    -- inverse linear map
+- `kwargs` -- additional arguments that are passed to the support function
+              algorithm
+
+### Output
+
+The support function in the given direction.
+If the direction has norm zero, the result depends on the wrapped set.
+
+### Notes
+
+If ``L = M^{-1}⋅S``, where ``M`` is a matrix and ``S`` is a convex set, it follows
+that ``ρ(d, L) = ρ((M^T)^{-1} d, S)`` for any direction ``d``.
+"""
+function ρ(d::AbstractVector, lm::InverseLinearMap)
+    return ρ(transpose(lm.M) \ d, lm.X)
+end
+
+
+"""
+    ∈(x::AbstractVector{N}, ilm::InverseLinearMap{N}) where {N<:Real}
+
+Check whether a given point is contained in an inverse linear map of a convex set.
+
+### Input
+
+- `x`   -- point/vector
+- `ilm` -- inverse linear map of a convex set
+
+### Output
+
+`true` iff ``x ∈ ilm``.
+
+### Algorithm
+
+This implementation does not explicitly invert the matrix since it uses the
+property ``x ∈ M^{-1}⋅S`` iff ``M⋅x ∈ S``..
+
+### Examples
+
+```jldoctest
+julia> ilm = LinearMap([0.5 0.0; 0.0 -0.5], BallInf([0., 0.], 1.));
+
+julia> [1.0, 1.0] ∈ ilm
+false
+julia> [0.1, 0.1] ∈ ilm
+true
+```
+"""
+function ∈(x::AbstractVector, lm::InverseLinearMap)
```

Comment:
```suggestion
function ∈(x::AbstractVector, ilm::InverseLinearMap)
```
and also in the body of the function